### PR TITLE
chore: release beta from latest version when publishing first time

### PR DESCRIPTION
**Pull request by ajays97**
Ticket: BG-77950

### CODE REVIEW

File: scripts/prepare-release.ts [hunk 1]
```diff
@@ -119,36 +119,35 @@ const incrementVersions = async (preid: string = 'beta') => {
...
-          JSON.stringify(json, null, 2) + '\n'
-        );
-        // since we're manually setting new versions, we must also reconcile all other lerna packages to now use the 'next' version for this module
-        lernaModuleLocations.forEach((otherModulePath) => {
-          // skip it for the current version
-          if (otherModulePath === modulePath) {
-            return;
+
+      const next = inc(tags[preid] || tags['latest'], 'prerelease', undefined, preid);
+      console.log(`Setting next version for ${json.name} to ${next}`);
```
Comment (sachushaji) [inline scripts/prepare-release.ts:124]: I can see that now some packages like web-demo, sdk-test etc which seems not published also are going through this logic flow. Probably the check was introduced to gate these internal packages probably. Lets wait for sdk teams feedback

**Review by LeoKotschenreuther — COMMENTED** [review]
Automatic comment to flush this PR from my review queue. 🚽🚽🚽
Please re-request another review from the respective team(s) when you are ready for reviews.

File: scripts/prepare-release.ts (hunk 1 above)
Comment (zahin-mohammad) [reply scripts/prepare-release.ts:124]: Web-demo and express should be ignored. They where deleted via github actions before this script is run.

Comment (mmcshinsky-bitgo) [reply scripts/prepare-release.ts:124]: Lerna should ignore those set the aren't set to public.

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**